allauth/socialaccount/adapter.py

```diff
@@ -116,7 +116,6 @@ def get_connect_redirect_url(self, request, socialaccount):
         Returns the default URL to redirect to after successfully
         connecting a social account.
         """
-        assert request.user.is_authenticated
```

Comment:
**Question**: Why is this `assert` being removed? Same question for the one below.